packages/rfw/example/wasm/lib/platforms/platform.dart

```diff
@@ -0,0 +1,13 @@
+import 'dart:typed_data';
+
+/// Platform agnostic API for retrieving data from the network. In this case,
+/// for RFW data and Wasm modules.
+abstract class Network {
+  Future<Uint8List> get(String url);
+}
+
+/// Platform agnostic API for loading Wasm modules and calling Wasm functions.
+abstract class Wasm {
+  Future<void> loadModule(String url);
+  dynamic call(String name, [List<Object?>? arguments]);
```

Comment:
can we call this `invoke` or `apply` or some such, rather than `call`? The word `call` has magical meaning in Dart.

Also we should probably make this return `Object?` rather than `dynamic`, to encourage good style.

Reply:
I just called it callFunction. Invoke/apply might sound like the object itself/`this` is the function. Also parametrized to add the expected return. Since through JS interop, even the int from C actually returns a double.